app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
 
 
-def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+def cache(func: Callable, cached_results: {} = None) -> Callable:
+    if cached_results is None:
+        cached_results = {}
+
+    def wrapper(*args) -> None:
```

Comment:
What would a better return type here? Decorated function will not necessarily return None